[prod]rebecca-rosenberg/update-browser-test-gv (#12500)
DataDog/documentation: content/en/synthetics/browser_tests/_index.md

{{< img src="synthetics/browser_tests/using_variables_browser.mp4" alt="Using Variables in Browser Tests" video="true"  width="80%" >}}

> You can also use global variables in your browser test's recorded steps by selecting `+ Variables` when recording the steps and selecting `Global Variable` from the Add Variable popup's dropdown. 

Reviewer: ```suggestion
To use global variables in your browser test's recorded steps, select **+ Variables** when recording your steps and select **Global Variable** from the dropdown menu. 
```
